box-tests.sh

```diff
@@ -28,11 +21,17 @@ runtests35-sqlite3 auth_tests
 runtests35-postgres auth_tests
 runtests35-mysql auth_tests
 
-runtests27-sqlite3 admin_widgets --selenium chrome
+runtests36-sqlite3-gis gis_tests
```

Comment:
Good idea - being consistent in the naming will help if I'm promoting the similarities between the environments. It'll also force me to cleanup the current ansible statements that generate the aliases which aren't pretty at all.